index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Web Development BootCamp</title>
+  </head>
+  <body>
+    <section class="main-section">
+      <nav class="mobile-menu">
+        <img src="images/enabled.jpg" alt="Enable icon" class="enabled-menu" />
+      </nav>
+      <nav class="mobile-menu-expanded">
+        <div class="closing-icon-container">
+          <img
+            src="images/ClosingIcon.png"
+            alt="Closing icon"
+            class="closing-icon"
+          />
+        </div>
+        <ul class="navigation">
+          <li class="home"><a href="index.html" class="hover">Home</a></li>
+          <li class="menu"><a href="about.html" class="hover">About</a></li>
+          <li class="menu"><a href="#bootcamp" class="hover">Bootcamp</a></li>
+          <li class="menu">
+            <a href="#instructors" class="hover">Instructors</a>
+          </li>
+          <li class="menu"><a href="#faq" class="hover">FAQ</a></li>
+          <li class="menu"><a href="#contact" class="hover">Contact</a></li>
+        </ul>
+      </nav>
+      <header class="desktop-menu">
+        <div class="pre-header">
+          <ion-icon name="logo-facebook" class="icon"></ion-icon>
+          <ion-icon name="logo-twitter" class="icon"></ion-icon>
+          <span class="tabs">English</span>
+          <span class="tabs">My Page</span>
+          <span class="tabs">Logout</span>
+        </div>
+        <div class="header">
+          <a href="index.html" class="logo-imgs"
+            ><img src="images/webdevsm.png" class="header-logo" alt="logo"
+          /></a>
+          <ul class="uo-list-header">
+            <li class="menu"><a href="about.html" class="hover">About</a></li>
+            <li class="menu">
+              <a href="#bootcamp" class="hover">Bootcamp</a>
+            </li>
+            <li class="menu">
+              <a href="#instructors" class="hover">Instructors</a>
+            </li>
+            <li class="menu"><a href="#faq" class="hover">FAQ</a></li>
+            <li class="menu"><a href="#contact" class="hover">Contact</a></li>
+
+            <li><a href="#" class="button-nav">Join</a></li>
+          </ul>
```

Comment:
A great way of creating an interactive section for authors just to fix minor things.
Such as text color and size in order to match the one from the template. As well as the subheading below the name to match the one from the template.

![Screenshot from 2023-08-02 15-14-39](https://github.com/memicelis/web-development-bootcamp/assets/118063058/f0defcbc-b8cd-4857-aa26-cf7cd199c3ac)

![Screenshot from 2023-08-02 15-14-47](https://github.com/memicelis/web-development-bootcamp/assets/118063058/31b291c5-253a-41ef-af79-eebcd8b9c183)